src/app/search/demos/onchange/search-onchange.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-search-onchange',
+  templateUrl: './search-onchange.component.html',
+  styles: [
+  ]
+})
+export class SearchOnChangeComponent implements OnInit {
+  onBlurSearchText = '';
+  OnChangeSearchText = ''
+  searchtext = '';
+  searchTextModel = '';
```

Comment:
Could we have an example with default value set for the model as well? Just updating this model value should be OK